docs/guidelines/CONTRIBUTING.md

```diff
@@ -5,6 +5,8 @@
 1. Fork this repository and clone it to your local machine
 2. Make sure you have `yarn` installed. If you don't, run `npm install -g yarn`
 3. Install all packages with the `yarn` command in the project root.
+4. Make a new firebase project. Add a web project to it.
```

Comment:
What do you mean by this?

Reply:
I meant to add a web app to the project, I can reword it